### PR TITLE
Analytics reports

### PR DESCRIPTION
**Pull request by shawnbot**
### 🚧 ⚠️  Work in progress ⚠️ 🚧 

This is a fix for 18F/web-design-standards#1579 that adds our quarterly high-level analytics report for sites that use the Standards.

- [x] Create analytics report table with custom columns
- [x] Convert about overview and research strategy pages back to Markdown
- [x] Import quarterly data files for 2016 Qs 2-4
  * This one is tricky because Tim Lowden reminded me that we're not supposed to be sharing analytics figures for specific domains without permission from the agencies that run them. I removed them manually for now.
- [x] ~Include the list of sites that use it here?~ We're not doing this
- [x] Remove broken link to sites that use the standards
- [x] Content review
- [x] Code review

### CODE REVIEW

**Comment by bradnunnally** [comment]
At this time I don't think we should include the master list of who's using the Standards. Because for some they link to a site that isn't using the Standards, because its being used on some internal site which we can't link to. This could lead to confusion for some. 

I imagine we will need to do an inventory of the public sites that we know are using them for sure and create a portfolio based on that. Heavier lift, but it'll lead to less confusion and we can present it really nicely.

**Comment by bradnunnally** [comment]
Based on the screenshot above, this looks good.

File: pages/about-our-work/overview.md [hunk 1]
```diff
@@ -4,21 +4,30 @@ layout: styleguide
...
+
+By accessing the Google Analytics data gathered for
+[analytics.usa.gov](https://analytics.usa.gov), we have aggregated a
+few data points to approximate how many users of federal websites
+have "experienced" the Draft U.S. Web Design Standards. We will
+update these numbers quarterly.
+
+{% include reports/analytics.html columns=page.columns %}
+
+#### Sites that use the standards
```
Comment (juliaelman) [inline pages/about-our-work/overview.md]: Remove this line.

File: pages/about-our-work/overview.md [hunk 2]
```diff
@@ -4,21 +4,30 @@ layout: styleguide
...
+
+### GitHub engagement
```
Comment (juliaelman) [inline pages/about-our-work/overview.md]: @bradnunnally we still cool with removing Github info from reporting? I vaguely remember us talking about doing that at one point.

**Comment by bradnunnally** [comment]
Yeah lets remove the github numbers.

**Comment by juliaelman** [comment]
@shawnbot TBH I am not a huge fan of squashing commits for smaller repos. It's nice to see folks thought processes, when they changed things, etc. 

However! For bigger and more complex repos squashing totally makes sense (e.g. https://github.com/18F/web-design-standards). Let's talk about that as we fine tune the release process.

**Pull request merged**